Sources/Widgets/Widgets3D/LineWidget/state.js

```diff
@@ -39,13 +50,14 @@ export default function generateState() {
       name: 'text',
       initialValues: {
         /* text is empty to set a text filed in the SVGLayer and to avoid
-         * displaying text before positionning the handles */
+         * displaying text before positioning the handles */
         text: '',
         visible: false,
         origin: [0, 0, 0],
       },
     })
-    .addField({ name: 'positionOnLine', initialValues: 0 })
-    .addField({ name: 'nbHandles', initialValues: 0 })
+    .addStateFromInstance({ name: 'positionOnLine', instance: linePosState })
+    .addField({ name: 'handle1FaceCamera', initialValue: true })
+    .addField({ name: 'handle2FaceCamera', initialValue: true })
```

Comment:
to move to arrow representation: forceFaceCamera (None let the behavior by default, true forces to face, false forces to not face)